assets/playground.html

```diff
@@ -618,6 +625,16 @@
                       </svg>
                     </div>
                   </template>
+                  <template
+                    x-if="message.state == 'succeed' && !!window.speechSynthesis">
+                    <div class="tts-message-btn" @click="handleTTSMessage(message.content)" title="TTS">
+                      <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" fill="#000000" viewBox="0 0 24 24" version="1.1" x="0px" y="0px"><title>icon/play-pause</title>
```

Comment:
> `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" fill="#000000" viewBox="0 0 24 24" version="1.1" x="0px" y="0px"><title>icon/play-pause</title>`

Two problems: 
1. The icon doesn't match the other icons.
![image](https://github.com/user-attachments/assets/4dc20d25-1dea-4f67-b275-08aa9aadfb93)

2. The `fill="#000000" will cause display issues under dark color schemes.
![image](https://github.com/user-attachments/assets/7ea521b4-6c92-4e07-a32a-9b0ea91ff4a0)

All the other icons are from Bootstrap Icons, I also recommend this one: https://icons.getbootstrap.com/icons/volume-up/

It is also recommended to refer to style of other svg icons.
https://github.com/sigoden/aichat/blob/80d3edbd43e874bb30decf1c5d13834a9a896007/assets/playground.html#L614